core_test/dune

```diff
@@ -29,9 +20,6 @@ let () = Printf.ksprintf Jbuild_plugin.V1.send {|
     lwt/link_test_main.exe
     ../switch/switch_main.exe
     ../cli/main.exe
-    basic-rpc-test.sh
```

Comment:
Did you intend to drop this?

Reply:
Yes, it's not needed since the dependency is inferred from the action.